libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -438,20 +438,21 @@ namespace SohImGui {
     }
 
 
-    void EnhancementCombobox(const char* name, const char* ComboArray[], uint8_t FirstTimeValue = 0){
-        if (FirstTimeValue <= 0){
+    void EnhancementCombobox(const char* name, std::vector<std::string> ComboArray, uint8_t FirstTimeValue = 0) {
```

Comment:
Avoid passing a vector by value to a function, since that will create a copy of the vector every time the function is invoked with an lvalue.